### DISCUSSION TITLE
How to configure timezone

### DISCUSSION THREAD

**Discussion by agentcobra** [dunglas/symfony-docker#647] [Q&A] [answered]
Hi everybody,
Where can I set the timezone configuration ?
Best regards
Agentcobra

**Comment by Trim** [comment, marked as answer]
Hello,

IIRC you can set it with the [date.timezone](https://www.php.net/manual/en/datetime.configuration.php#ini.date.timezone) PHP setting. You can set it in the `frankenphp/conf.d/app.ini` configuration file in a symfony-docker project.

**Reply by agentcobra** [reply]
thanks for your answer. Do I need to rebuild if I change this configuration?

**Reply by Trim** [reply]
For local development, there's no need to rebuild, because the `compose.override.yaml` file creates a volume to share directly files between the host (your computer) and the container.

For other environments (like `production`), yes you need to rebuild the image and deploy it again.

**Reply by Trim** [reply]
> For local development, there's no need to rebuild, because the `compose.override.yaml` file creates a volume to share directly files between the host (your computer) and the container.
> 
> For other environments (like `production`), yes you need to rebuild the image and deploy it again.

Sorry, I forgot the `app.ini` files are not synchronized with the volume, so you are right: every time you want to modify a PHP setting in `app.ini` files you have to rebuild the container image.

**Comment by treeindark** [comment]
```
environment:
    TZ: Europe/Paris
```
